apps/server/src/api-data/excel/__tests__/excel.parser.test.ts

```diff
@@ -168,10 +168,41 @@ describe('parseExcel()', () => {
       timerType: 'timer type',
     };
     const result = parseExcel(testdata, {}, 'testSheet', importMap);
-    const firstEvent = result.rundown.entries[result.rundown.order[0]];
+    const firstGroup = result.rundown.entries[result.rundown.order[0]];
+
+    expect(result.rundown.order.length).toBe(1);
+    expect(result.rundown.flatOrder.length).toBe(2);
+    expect((firstGroup as OntimeEvent).type).toBe(SupportedEntry.Group);
+  });
+
+  it('places event between groups inside the group', () => {
```

Comment:
suggestion: could you add the `duration` property here to make sure it gets imported to `targetDuration`?